.nvmrc

```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
 14.18.2
+=======
+14
+>>>>>>> 8e508d97 (chore(repo): update node version to 14)
```

Comment:
Please remove .nvmrc changes.

Please look here https://github.com/betaflight/betaflight/wiki/Betaflight-Contribution#squash-your-commits how to squash your commits

Reply:
i'm battling github right now to figure out how to squash and stop messing stuff up

Reply:
Been there done that. Spent hours to get a grasp. So hope my document will guide any developers as these are my working notes.

Reply:
Actually days 😂